--- a/shim.c	2024-01-22 20:18:05.000000000 +0100
+++ b/shim.c	2024-06-17 07:21:17.813163782 +0200
@@ -1260,7 +1260,8 @@
 	// If the filename is invalid, or the file does not exist,
 	// just fallback to the default loader.
 	if (!use_fb && (efi_status == EFI_INVALID_PARAMETER ||
-	                efi_status == EFI_NOT_FOUND)) {
+	                efi_status == EFI_NOT_FOUND || 
+			efi_status == EFI_TFTP_ERROR)) {
 		console_print(
 			L"start_image() returned %r, falling back to default loader\n",
 			efi_status);
