diff --git a/Make.defaults b/Make.defaults
index 18677da..73d306f 100644
--- a/Make.defaults
+++ b/Make.defaults
@@ -28,7 +28,7 @@ DATATARGETDIR	?= $(datadir)/$(PKGNAME)/$(VERSION)$(DASHRELEASE)/$(ARCH_SUFFIX)/
 DEBUGINFO	?= $(prefix)/lib/debug/
 DEBUGSOURCE	?= $(prefix)/src/debug/
 OSLABEL		?= $(EFIDIR)
-DEFAULT_LOADER	?= \\\\grub$(ARCH_SUFFIX).efi
+DEFAULT_LOADER	?= \\\\opsi-netboot.efi
 DASHJ		?= -j$(shell echo $$(($$(grep -c "^model name" /proc/cpuinfo) + 1)))
 
 ARCH		?= $(shell $(CC) -dumpmachine | cut -f1 -d- | sed s,i[3456789]86,ia32,)
diff --git a/shim.h b/shim.h
index 69442da..d22113d 100644
--- a/shim.h
+++ b/shim.h
@@ -70,10 +70,10 @@
 
 #ifdef __x86_64__
 #ifndef DEFAULT_LOADER
-#define DEFAULT_LOADER L"\\grubx64.efi"
+#define DEFAULT_LOADER L"\\opsi-netboot.efi"
 #endif
 #ifndef DEFAULT_LOADER_CHAR
-#define DEFAULT_LOADER_CHAR "\\grubx64.efi"
+#define DEFAULT_LOADER_CHAR "\\opsi-netboot.efi"
 #endif
 #ifndef EFI_ARCH
 #define EFI_ARCH L"x64"
